pkg/network/driver/ethtool/ethtool.go

```diff
@@ -10,13 +29,29 @@ import (
 	"kubevirt.io/client-go/log"
 )
 
+type EthtoolH struct{}
+
+var ethtoolH EthtoolH
+
 const (
 	SIOCETHTOOL     = 0x8946     // linux/sockios.h
 	ETHTOOL_GTXCSUM = 0x00000016 // linux/ethtool.h
 	ETHTOOL_STXCSUM = 0x00000017 // linux/ethtool.h
 	IFNAMSIZ        = 16         // linux/if.h
 )
 
+func Ethtool() EthtoolH {
+	return ethtoolH
+}
+
+func (e EthtoolH) TXChecksumOff(name string) error {
+	return EthtoolTXOff(name)
```

Comment:
shouldn't we make `EthtoolTXOff` private now?

Reply:
Yes, it should be made private with the removal of `DisableTXOffloadChecksum`.
I will change it in the next update of the PR.

Reply:
This has been resolved already in the `network: Remove code that has become unused` commit.
Until then, the mentioned method is used.